governance.md

```diff
@@ -153,49 +220,127 @@ are enumerated below.
 
 ## Other duty-specific github roles:
 - [**Github Org Owner**](https://github.com/orgs/kubernetes/people?utf8=%E2%9C%93&query=%20role%3Aowner):
-  can create repos, do ~any github action; the number of
+  can create repos, and do any github action; the number of
   owners shouldn't scale with the organization's growth, O(1), and optimally it
   should be less than 20 people who are very familiar with project workings and
   distributed across a few time zones and organizations The other repos will
   have distinct sets of people filling some of the above roles, also.
 
+## Procedural roles
+
+- **Champion**
+  - A Champion is the primary point of contact for guiding someone through a process, such
+    as [creating a new incubator repo](incubator.md) or becoming a maintainer
+  - The majority of the mentorship, review, and advice regarding Kubernetes community norms and
+    processes will come from the Champion
+  - Potential Champions come from a group of existing Kubernetes contributors, such as REVIEWERS
+    or MAINTAINERS -- which group depends on the particular process 
+- **Sponsor**
+  - A Sponsor is an approver for initiating a process, such as creating a new incubator repo or
+    adding a new maintainer
+  - Potential Sponsors come from a very small set of senior Kubernetes contributors (typically LEADS)
```

Comment:
this mentions `LEADS` but that section of the google doc has not been brought over here...